content/talks.njk

```diff
@@ -0,0 +1,57 @@
+---
+title: Talks & Workshops
+sub: Bring us to your conference or browse past talks
+image:
+  src: talks/miriam.jpg
+  alt: Miriam Suzanne speaking at a conference
+  position: left center
+action:
+  text: Hire us for conference talks »
+  url: /contact/
+summary: |
+  **We provide award-winning conference talks & workshops**
+  to help you create resilient web applications
+  and design systems --
+  improving team process, communication,
+  site performance, and accessibility.
+  Missed a conference?
+  Dive into the posts below
+  for the latest videos & slides,
+  and upcoming conference dates.
+---
+
+{% import 'quotes.macros.njk' as quotes %}
+{% import 'layout.macros.njk' as layout %}
+{% import 'talks.macros.njk' as talks %}
+{% import 'post.macros.njk' as post %}
+
+{{ quotes.find(
+  collections.all,
+  slugs=['spot-on', 'popular-tutorial']
+) }}
+
+{{ talks.section(
+  title='Upcoming Events',
+  events=collections.all | getEvents | getFuture,
+  all=false,
+  by_bird=collections
+) }}
+
+{{ layout.title('Talks & Workshops') }}
+{% call layout.grid('training') %}
+  {% for item in collections._talk | eventSort | reverse %}
+    <article {{ post.aria_label(item) }} class="training-item">
+      {{ post.hero(item.data.image, 'post') }}
+      <header class="training-header">
+        {{ post.banner(item) }}
+        {{ post.subtitle(item.data.sub) }}
+        {{ post.byline(item, collections, dateformat='range') }}
```

Comment:
After reviewing the comments in the XD file, I am assuming we went this direction in order to not show the "published date" as a post would in the byline. I still think it could be more clear. Maybe a pipe or other punctuation after the name, then maybe some words that address the Talk or Workshop was created in 20xx (and not the human)?

Reply:
Yeah, it makes sense once you realize it means "talk first given in 20xx." Not sure what a good fix would be without being to verbose. "First given by X in 20xx"?